### PR TITLE
Grammar and wording fixes for 1.20 release notes

### PR DESCRIPTION
**Pull request by laverya**

### CODE REVIEW

File: content/release-notes/1.20.0.md [hunk 1]
```diff
@@ -22,24 +22,24 @@ kubernetes: "1.17, 1.18, and 1.19"
...
+* Port forwarding no longer uses the `app.k8s.io/v1beta1` Application spec and has been aligned with the [documentation](/vendor/config/dashboard-buttons/#provide-a-kots-application-spec).
 * [SupportBundle](https://troubleshoot.sh/docs/support-bundle/collecting/) and [Preflight](https://troubleshoot.sh/docs/preflight/introduction/) specs are no longer served on unauthenticated endpoints and instead are provided to [support-bundle and preflight binaries](https://troubleshoot.sh/docs/#installation) via [secrets](/kotsadm/troubleshooting/support-bundle/).
-* Version check now correctly reports current application version and adds information about KOTS Admin Console version, Kubernetes version, app status, etc.
+* The version check endpoint now correctly reports the current application version and adds information about KOTS Admin Console version, Kubernetes version, app status, etc.
 {{</changes>}}
 
 {{<fixes>}}
-* Airgap bundles can now only be installed if the app matches the license.
+* When an airgap bundle that does not match the installed license is uploaded, show an error message that explicitly says this instead of "failed to verify bundle signature".
```
Comment (laverya) [inline content/release-notes/1.20.0.md:32]: It's always been the case that you'll fail to install bundles from different apps than the license
It's just the error message was horrible: https://github.com/replicatedhq/kots/pull/1096

Comment (markpundsack) [inline content/release-notes/1.20.0.md:32]:
```suggestion
* Added an explicit error message when an uploaded airgap bundle does not match the installed license.
```

File: content/release-notes/1.20.0.md [hunk 2]
```diff
@@ -22,24 +22,24 @@ kubernetes: "1.17, 1.18, and 1.19"
...
-* Fixed a bug that caused [GitOps](/kotsadm/gitops/single-app-workflows/) to fail to clone empty repo on initial setup.
-* Fixed a troubleshoot bug where the result of a [secret collector](https://troubleshoot.sh/docs/collect/secret/) would have missing key.
-* Fixed a troubleshoot bug where a [run collector](https://troubleshoot.sh/docs/collect/run/) would hang if Pod is in state ImagePullBackoff.
-* Fixed a troubleshoot bug that caused [ingress analyzer](https://troubleshoot.sh/docs/analyze/ingress/) to not produce a result.
-* Fixed a troubleshoot bug that caused [database version analyzer](https://troubleshoot.sh/explore?tag=database) to fail on versions that aren't strictly semver conformant.
-* Fixed a troubleshoot bug that caused [distribution analyzer](https://troubleshoot.sh/docs/analyze/distribution/) to not recognize IBM and minikube distributions.
-* Fixed reporting of "released on" date to use date of promotion to channel rather than date download.
+* Fixed a bug that caused [GitOps](/kotsadm/gitops/single-app-workflows/) to fail to clone an empty repo on initial setup.
```
Comment (markpundsack) [inline content/release-notes/1.20.0.md:38]:
```suggestion
* Fixed a bug that caused [GitOps](/kotsadm/gitops/single-app-workflows/) to fail on initial setup if the repo is empty.
```

Comment (markpundsack) [reply content/release-notes/1.20.0.md:38]: Is it really when it's "empty" or "unitialized"?

File: content/release-notes/1.20.0.md [hunk 3]
```diff
@@ -22,24 +22,24 @@ kubernetes: "1.17, 1.18, and 1.19"
...
+* Fixed a troubleshoot bug where the result of a [secret collector](https://troubleshoot.sh/docs/collect/secret/) would have a missing key.
+* Fixed a troubleshoot bug where a [run collector](https://troubleshoot.sh/docs/collect/run/) would hang if the Pod was in state ImagePullBackoff.
```
Comment (markpundsack) [inline content/release-notes/1.20.0.md:40]:
```suggestion
* Fixed a troubleshoot bug where a [run collector](https://troubleshoot.sh/docs/collect/run/) would hang if the Pod was in the `ImagePullBackoff` state.
```

File: content/release-notes/1.20.0.md (hunk 2 above)
Comment (laverya) [reply content/release-notes/1.20.0.md]: I have no idea
I know the grammar/wording was off

**Review by markpundsack — APPROVED** [review]

**Pull request merged**